### PR TITLE
fix(TDI-48434): Avoid NPE for the old xlsx format

### PR DESCRIPTION
**Pull request by dmytro-sylaiev**
**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by wwang-talend — APPROVED** [review]

**Review by dmytrogrygorenko — DISMISSED** [review]
LGTM

File: main/plugins/org.talend.designer.components.localprovider/components/tFileInputExcel/tFileInputExcel_java.xml [hunk 1]
```diff
@@ -177,7 +177,7 @@
 	<CODEGENERATION>
 		<IMPORTS>
 			<IMPORT NAME="Java_Excel" MODULE="jxl.jar" MVN="mvn:org.talend.libraries/jxl/6.0.0"  UrlPath="platform:/plugin/org.talend.libraries.jexcel/lib/jxl.jar" REQUIRED_IF="(VERSION_2007 == 'false')" />
-			<IMPORT NAME="simpleexcel" MODULE="simpleexcel-2.5-20201119.jar" MVN="mvn:org.talend.components/simpleexcel/2.5-20201119" UrlPath="platform:/plugin/org.talend.libraries.excel/lib/simpleexcel-2.5-20201119.jar" REQUIRED_IF="(VERSION_2007 == 'true') AND GENERATION_MODE == 'EVENT_MODE')" />
```
Comment (ozhelezniak-talend) [inline main/plugins/org.talend.designer.components.localprovider/components/tFileInputExcel/tFileInputExcel_java.xml:180]: :thumbsup:

File: main/plugins/org.talend.libraries.excel/.classpath [hunk 2]
```diff
@@ -8,7 +8,6 @@
 	<classpathentry exported="true" kind="lib" path="lib/poi-ooxml-4.1.2-20200903124306_modified_talend.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/poi-ooxml-schemas-4.1.2-20200903124306_modified_talend.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/poi-scratchpad-5.2.2.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/simpleexcel-2.5-20201119.jar"/>
```
Comment (ozhelezniak-talend) [inline main/plugins/org.talend.libraries.excel/.classpath:11]: Were we the only one who use this library?

Comment (dmytro-sylaiev) [reply main/plugins/org.talend.libraries.excel/.classpath:11]: https://github.com/search?q=org%3ATalend+simpleexcel&type=code
looks like that

**Review by ozhelezniak-talend — APPROVED** [review]
lgtm

**Pull request merged**